### DISCUSSION TITLE
Module for topic/subscription - Azure ServiceBus Namespace

### DISCUSSION THREAD

**Discussion by amullersusu** [Azure/ResourceModules#3071] [General]
For my project I need to add a subscription in my topic for Azure ServiceBus Namespace, and would like to add this in the parameter file when deploying via AzureDevops yml pipeline. 

It looks like this is not currently possible as I need to add the Subscription as an module in order to reference and deploy it. Is this something that is being worked on?

**Comment by eriqua** [comment]
Hi @amullersusu, thanks for opening this discussion and sorry for the late reply. Discussions are currently not monitored promptly and are planned to be disabled in favor of the new issue template [Questions/Feedback].

I'm converting this discussion to an issue in order for the team to be able to triage it.

Please refer to issue #3145 going forward.
